auth/server/src/main/java/com/seagates3/dao/ldap/PolicyLdapStore.java

```diff
@@ -148,22 +149,48 @@ Policy find(Object policyArnStr,
   }
 
  public
-  List<Policy> findAll(Object accountObj) throws DataAccessException {
+  List<Policy> findAll(Object accountObj,
+                       Object parametersObj) throws DataAccessException {
     Account account = (Account)accountObj;
-    String[] attrs = {
-        LDAPUtils.POLICY_ID,                 LDAPUtils.PATH,
-        LDAPUtils.POLICY_CREATE_DATE,        LDAPUtils.POLICY_UPDATE_DATE,
-        LDAPUtils.DEFAULT_VERSION_ID,        LDAPUtils.POLICY_DOC,
-        LDAPUtils.POLICY_NAME,               LDAPUtils.IS_POLICY_ATTACHABLE,
-        LDAPUtils.POLICY_ARN,                LDAPUtils.POLICY_ATTACHMENT_COUNT,
-        LDAPUtils.POLICY_PERMISSION_BOUNDARY};
+    Map<String, Object> parameters = (Map<String, Object>)parametersObj;
```

Comment:
parameter is a very generic term. We should avoid it

Reply:
fixed